### ISSUE TITLE
PvP stats always null

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SourceCodeDude/n-battle-net-api#18]
```
What steps will reproduce the problem?
1. add CharacterFields.PVP flag (or anything for that matter) in GetCharacter
2. Character.PvP is always = null

What is the expected output? What do you see instead?
It should have the PvP stats, arena teams, etc.

What version of the product are you using? On what operating system?
0.5.0.0 on Windows 7

Please provide any additional information below.
Could it have anything to do with this line in CharacterQuery.cs?:

if ((Fields & CharacterFields.PVP) == CharacterFields.Guild) args.Add("pvp");


```

Original issue reported on code.google.com by `je...@the650.org` on 8 Sep 2011 at 12:17

**Comment by GoogleCodeExporter** [comment]
```
Yep
```

Original comment by `shimmanning@gmail.com` on 12 Sep 2011 at 11:55
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in 0.6.0.0
```

Original comment by `shimmanning@gmail.com` on 24 Sep 2011 at 1:47
- Changed state: **Fixed**
